editions/tw5.com/tiddlers/saving/Saving with TiddlyFox on Android.tid

```diff
@@ -1,12 +1,12 @@
 created: 20140103134551508
-modified: 20171113131640857
+modified: 20230803051340676
 tags: [[Saving with TiddlyFox]]
 title: Saving with TiddlyFox on Android
 type: text/vnd.tiddlywiki
 
```

Comment:
Sadly this tiddler also needs to start with `<<.deprecated-since "FireFox 57" "Saving">>`

There is no way to save with FF for Android using browser addOns at the moment. As soon at it will be possible I'll probably make the file-saver AddOn compatible. Till then it does not work anymore.